src/manager/source_driver_ros1.hpp

```diff
@@ -233,7 +233,7 @@ inline sensor_msgs::PointCloud2 SourceDriver::ToRosMsg(const LidarDecodedFrame<L
   }
   printf("frame:%d points:%u packet:%d start time:%lf end time:%lf\n",frame.frame_index, frame.points_num, frame.packet_num, frame.points[0].timestamp, frame.points[frame.points_num - 1].timestamp) ;
   // ros_msg.header.seq = s;
-  ros_msg.header.stamp = ros::Time().fromSec(frame.points[0].timestamp);
+  ros_msg.header.stamp = ros::Time::now();
```

Comment:
I guess the timestamp of `frame.points[0]` comes from the hardware and it is not stable or has some offsets. Is my understanding correct?
If so, could you add some comments about why we want to use `ros::Time::now()` here?

Reply:
Yes.
frame.points[0] has offsets, it seem the duration from the process start?
I will add comments.

Reply:
> I will add comments.

Thanks 👍

Reply:
I added the comment.